extensions/wikia/AnalyticsEngine/AnalyticsProviderAmazonMatch.php

```diff
@@ -13,22 +13,19 @@ public static function isEnabled() {
 	}
 
 	private function getIntegrationScript( $moduleName, $instantGlobalName ) {
-		$moduleName1 = json_encode( 'ext.wikia.adEngine.lookup.' . $moduleName );
-		$moduleName2 = json_encode( 'ext.wikia.adEngine.' . $moduleName );
+		$moduleName = json_encode( 'ext.wikia.adEngine.lookup.' . $moduleName );
 		$instantGlobalName = json_encode( $instantGlobalName );
 
 		$code = <<< CODE
 	require([
 		"wikia.geo",
 		"wikia.instantGlobals",
-		require.optional($moduleName1), // new name
-		require.optional($moduleName2)  // old name
-	], function (geo, globals, amazon1, amazon2) {
-		var ac = globals[$instantGlobalName],
-			amazon = amazon1 || amazon2;
+		require.optional($moduleName),
+	], function (geo, globals, amazon) {
+		var ac = globals[$instantGlobalName];
 
 		if (geo.isProperGeo(ac)) {
-			amazon.call();
+			amazon.call('oasis');
```

Comment:
Thanks for cleaning up! However, I'm afraid that part of this code can be cached on varnishes and we need to make sure it will still work if `amazon.call()` will be fired.